scalafix-tests/unit/src/test/scala/scalafix/tests/cli/CliSyntacticSuite.scala

```diff
@@ -293,7 +293,7 @@ class CliSyntacticSuite extends BaseCliSuite {
     outputAssert = { out =>
       assert(
         out.endsWith(
-          """|a.scala:1:1: error: expected class or object definition identifier
+          """|a.scala:1:1: error: illegal start of definition identifier
```

Comment:
this looks like a cosmetic regression caused by the Ammonite parsing changes https://github.com/scalameta/scalameta/releases/tag/v4.4.35 cc @kitbellew

Reply:
By "Ammonite parsing changes", I meant the parser refactoring changes listed to better support Ammonite parsing. Anyway, the error message is still actionable, so I didn't feel like opening an issue to track the "regression" was required.